src/puppetlabs/puppetdb/query/paging.clj

```diff
@@ -94,14 +91,12 @@
   that satisfy `order-by-expr?`"
   [order_by]
   {:post [(every? order-by-expr? %)]}
-  (when-let [bad-order-by (some
-                           (fn [x] (when-not (contains? x :field) x))
-                           order_by)]
+  (when (some (fn [x] (when-not (contains? x :field) x))
+              order_by)
     (throw (IllegalArgumentException.
             (tru "Illegal value ''{0}'' in :order_by; missing required key 'field'."))))
```

Comment:
This should be changed. We do need the value from the original `when-let`, we just mistakenly weren't using it. The `tru` call here is missing its `0` input parameter. Our error messages must have been missing this value for a while and no one noticed. It should be something like this:
```
  (when-let [bad-order-by (some #(when-not (contains? % :field) %) order_by)]
    (throw (IllegalArgumentException.
            (tru "Illegal value ''{0}'' in :order_by; missing required key 'field'."
                 bad-order-by))))
```